bridge/telegram/handlers.go

```diff
@@ -537,34 +537,37 @@ func (b *Btelegram) handleEntities(rmsg *config.Message, message *tgbotapi.Messa
 			}
 			rmsg.Text = string(utf16.Decode(asRunes[:offset+e.Length])) + " (" + url.String() + ")" + string(utf16.Decode(asRunes[offset+e.Length:]))
 			indexMovedBy += len(url.String()) + 3
+			prevLinkOffset = e.Offset
 		}
 
-		if e.Type == "code" {
-			offset := e.Offset + indexMovedBy
-			rmsg.Text = string(utf16.Decode(asRunes[:offset])) + "`" + string(utf16.Decode(asRunes[offset:offset+e.Length])) + "`" + string(utf16.Decode(asRunes[offset+e.Length:]))
-			indexMovedBy += 2
-		}
+		if e.Offset != prevLinkOffset {
```

Comment:
```suggestion
		if e.Offset == prevLinkOffset {
			continue
		}
```

To keep the linter happy and make it a bit less complex